### DISCUSSION TITLE
Starting Perl

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#58] [Q&A] [answered]
What would you recommend to a person who wants to learn Perl?

**Comment by Stagyrite** [comment, marked as answer]
If you need to learn Perl, a full academic course is what you need. In the particular case of a person who likes to learn Perl, I'd recommend installing a Linux operating system. I'd also give her some books about Perl and Raku to read. One can also join an online paid course.
My approach is historical because I also try the Perl 1 compiler, although I don't recommend it to everyone. It's also related to learning a foreign language because I got books in German. I want to learn German, and therefore I read them.
